front/src/common/redux/actions/groupListAction.js

```diff
@@ -0,0 +1,8 @@
+import { CURRENT_GROUP } from './types';
+
+export const setCurrentGroup = (data) => {
+  return {
+    type: CURRENT_GROUP,
```

Comment:
It is better to use a verb for the name of this action type

Reply:
done